app/Models/FormAuth.php

```diff
@@ -7,9 +7,9 @@ public static function checkCredentials($username, $hash, $nonce, $challenge) {
 				!ctype_graph($challenge) ||
 				!ctype_alnum($nonce)) {
 			Minz_Log::debug('Invalid credential parameters:' .
-			                ' user=' . $username .
-			                ' challenge=' . $challenge .
-			                ' nonce=' . $nonce);
+				' user=' . $username .
+				' challenge=' . $challenge .
+				' nonce=' . $nonce);
```

Comment:
See aforementioned comment

Reply:
That is indeed an obvious case :-)

Reply:
https://github.com/FreshRSS/FreshRSS/pull/3488/commits/c86067c0de096057fa1e494d5fa947e93e09b81f